components/02-molecules/components/accordion-plus-minus.html

```diff
@@ -7,33 +7,7 @@
       {{> @icon-plus-minus active="false" }}
     </div>
     <div id="panel-1-a" class="content">
-      <div class="checkbox-group" role="group">
-        <div class="checkbox-block-accordion">
```

Comment:
where did all the options go?

Reply:
@akegan so these were for the old version of the pattern library component where you could actually click to select/deselect the headers. Once I implemented it in webapp the functionality to select/deselect was so different I removed the functionality from the pattern library because it just complicated things.

So now all of these options under non-active headers are never able to be seen, since it's hard-coded which header is active. I never removed them in my previous PRs so I just removed them in this one.

Reply:
ok thanks for the explanation!